Table.java

adammantadakis opened the pull request.

adammantadakis committed:
Table.java

chrisk44 commented on Server/Table.java:
```diff
@@ -0,0 +1,118 @@
+	private int floor;
+	private List<Order> order;
+	private double balance;
+	public enum status{ free, reserved, calling, to_clean };
```
`public enum Status { FREE, RESERVED, PAID, CALLING, TO_CLEAN, NORMAL, TAKEN };`
`private Status status = FREE;`

chrisk44 commented on Server/Table.java:
```diff
@@ -0,0 +1,118 @@
+	private LatLng lat_lng;
+	private int seats;
+	private int floor;
+	private List<Order> order;
```
Antikeimeno Order anti gia lista

adammantadakis committed:
Corrections and changes

Made order an object instead of a List.
Corrected onCall().
Finished onOrderPaid(), setTaken() and isAvailable().
onTopologyEdit() still needs work.

anthoulab commented on Server/Table.java:
```diff
@@ -0,0 +1,117 @@
+	}
+
+
+	public void setReserved()
```
boolean

anthoulab committed:
Update Table.java
